.circleci/config.yml

```
@@ -0,0 +1,32 @@
+version: 2
+jobs:
+  build-and-test:
```

```suggestion
  build:
```
Workflow is not needed here.